### PR TITLE
Homepage static content updates

### PR DESCRIPTION
**Pull request by tudorraul**

### CODE REVIEW

**Comment by rtyley** [comment]
Regarding the campaign marker links, doing a quick grep with:

```
git grep 'INTCMP.*"'
```

on  https://github.com/guardian/subscriptions-frontend & https://github.com/guardian/subscriptions, the campaign codes look to match, so :+1:

**Comment by tudorraul** [comment]
what about the krux code?

**Comment by rtyley** [comment]
I don't know much about Krux, so can't really speak to that...

File: app/views/fragments/analytics/krux.scala.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 <script class="kxct" data-id="Jglpp88U" data-timing="async" data-version="1.9" type="text/javascript">
-    window.Krux || ((Krux=function() { Krux.q.push(arguments) }).q=[]);
-    (function(){
-        var k=document.createElement('script');k.type='text/javascript';k.async=true;
-        var m,src=(m=location.href.match(/\bkxsrc=([^&]+)/))&&decodeURIComponent(m[1]);
-        k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test(src) ? src : src === "disable" ? "" : (location.protocol==="https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid=Jglpp88U";
-        var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(k,s);
-    }());
-</script>
+  window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);
```
Comment (davidrapson) [inline app/views/fragments/analytics/krux.scala.html]: @tudorraul Bit late on the feedback here as it's merged, but if the new subscriptions is going to be over HTTPS you'll want to make sure you check for mixed-content warnings from Krux as we have had a lot of problems with that on Membership. It's OK once configured, but something to be aware of.

File: app/views/fragments/head.scala.html [hunk 2]
```diff
@@ -24,4 +24,4 @@
     <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicons/32x32.ico")"/>
 }
 
-@* <script src="//cdn.optimizely.com/js/293455139.js"></script> *@
+<script src="//cdn.optimizely.com/js/293455139.js"></script>
```
Comment (davidrapson) [inline app/views/fragments/head.scala.html]: As this is in the head (I get why A/B testing scripts are in the head) you should definitely add `async defer` attributes to this as in the currently could cause SPOF errors. See https://github.com/guardian/membership-frontend/pull/421

File: app/views/fragments/analytics/krux.scala.html (hunk 1 above)
Comment (tudorraul) [inline app/views/fragments/analytics/krux.scala.html]: Thanks for the heads up. It looks like all krux calls are over https which return 200. Also, there is no security warning from the browser. Thanks for flagging.

**Pull request merged**